gtk+2 fails with Xcode11 (cups)

gtk+2-2.18.9 fails with this error:
```
/bin/sh ../../../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -I../../../gtk -I../../../gtk -I../../../gdk -I../../../gdk  -DGTK_PRINT_BACKEND_ENABLE_UNSUPPORTED -I/opt/sw/include/glib-2.0 -I/opt/sw/lib/glib-2.0/include -I/opt/sw/include/pango-1.0 -I/opt/sw/include/atk-1.0 -I/opt/sw/include/cairo -I/opt/sw/include/pango-1.0 -I/opt/sw/include/freetype2 -I/opt/sw/include -I/opt/X11/include    -DG_DISABLE_CAST_CHECKS -DDISABLE_VISIBILITY  -D_IPP_PRIVATE_STRUCTURES=1 -I/opt/sw/include -DG_DISABLE_SINGLE_INCLUDES -DATK_DISABLE_SINGLE_INCLUDES -DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES -I/opt/X11/include  -DGDK_PIXBUF_DISABLE_DEPRECATED -Os -Wall -MT gtkprintbackendcups.lo -MD -MP -MF .deps/gtkprintbackendcups.Tpo -c -o gtkprintbackendcups.lo gtkprintbackendcups.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -I../../../gtk -I../../../gtk -I../../../gdk -I../../../gdk -DGTK_PRINT_BACKEND_ENABLE_UNSUPPORTED -I/opt/sw/include/glib-2.0 -I/opt/sw/lib/glib-2.0/include -I/opt/sw/include/pango-1.0 -I/opt/sw/include/atk-1.0 -I/opt/sw/include/cairo -I/opt/sw/include/pango-1.0 -I/opt/sw/include/freetype2 -I/opt/sw/include -I/opt/X11/include -DG_DISABLE_CAST_CHECKS -DDISABLE_VISIBILITY -D_IPP_PRIVATE_STRUCTURES=1 -I/opt/sw/include -DG_DISABLE_SINGLE_INCLUDES -DATK_DISABLE_SINGLE_INCLUDES -DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES -I/opt/X11/include -DGDK_PIXBUF_DISABLE_DEPRECATED -Os -Wall -MT gtkprintbackendcups.lo -MD -MP -MF .deps/gtkprintbackendcups.Tpo -c gtkprintbackendcups.c  -fno-common -DPIC -o .libs/gtkprintbackendcups.o
gtkprintbackendcups.c:352:18: warning: 'ppdFindAttr' is deprecated: first deprecated in macOS 10.8 - Use
      cupsCopyDestInfo and friends instead. [-Wdeprecated-declarations]
      ppd_attr = ppdFindAttr (ppd_file, "LanguageLevel", NULL);
                 ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ppd.h:398:20: note: 
      'ppdFindAttr' has been explicitly marked deprecated here
extern ppd_attr_t       *ppdFindAttr(ppd_file_t *ppd, const char *name,
                         ^
gtkprintbackendcups.c:359:26: warning: 'ppdFindAttr' is deprecated: first deprecated in macOS 10.8 - Use
      cupsCopyDestInfo and friends instead. [-Wdeprecated-declarations]
          ppd_attr_res = ppdFindAttr (ppd_file, "DefaultResolution", NULL);
                         ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ppd.h:398:20: note: 
      'ppdFindAttr' has been explicitly marked deprecated here
extern ppd_attr_t       *ppdFindAttr(ppd_file_t *ppd, const char *name,
                         ^
gtkprintbackendcups.c:380:34: warning: 'ppdFindAttr' is deprecated: first deprecated in macOS 10.8 - Use
      cupsCopyDestInfo and friends instead. [-Wdeprecated-declarations]
      ppd_attr_res_screen_freq = ppdFindAttr (ppd_file, "ResScreenFreq", res_string);
                                 ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ppd.h:398:20: note: 
      'ppdFindAttr' has been explicitly marked deprecated here
extern ppd_attr_t       *ppdFindAttr(ppd_file_t *ppd, const char *name,
                         ^
gtkprintbackendcups.c:388:38: warning: 'ppdFindAttr' is deprecated: first deprecated in macOS 10.8 - Use
      cupsCopyDestInfo and friends instead. [-Wdeprecated-declarations]
          ppd_attr_res_screen_freq = ppdFindAttr (ppd_file, "ResScreenFreq", res_string);
                                     ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ppd.h:398:20: note: 
      'ppdFindAttr' has been explicitly marked deprecated here
extern ppd_attr_t       *ppdFindAttr(ppd_file_t *ppd, const char *name,
                         ^
gtkprintbackendcups.c:392:30: warning: 'ppdFindAttr' is deprecated: first deprecated in macOS 10.8 - Use
      cupsCopyDestInfo and friends instead. [-Wdeprecated-declarations]
      ppd_attr_screen_freq = ppdFindAttr (ppd_file, "ScreenFreq", NULL);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ppd.h:398:20: note: 
      'ppdFindAttr' has been explicitly marked deprecated here
extern ppd_attr_t       *ppdFindAttr(ppd_file_t *ppd, const char *name,
                         ^
gtkprintbackendcups.c:460:15: error: incomplete definition of type 'struct _ipp_attribute_s'
        job_id = attr->values[0].integer;
                 ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:845:45: error: incomplete definition of type 'struct _ipp_s'
      switch (dispatch->request->ipp_request->request.op.operation_id)
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:731:16: note: 
      forward declaration of 'struct _ipp_s'
typedef struct _ipp_s ipp_t;            /**** IPP request/response data ****/
               ^
gtkprintbackendcups.c:964:45: warning: passing 'gchar **' (aka 'char **') to parameter of type
      'const char *const *' discards qualifiers in nested pointer types
      [-Wincompatible-pointer-types-discards-qualifiers]
                                            dispatch->request->auth_info);
                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
./gtkcupsutils.h:170:32: note: passing argument to parameter 'values' here
                                                            const char * const *values);
                                                                                ^
gtkprintbackendcups.c:1450:23: error: incomplete definition of type 'struct _ipp_s'
  for (attr = response->attrs; attr != NULL; attr = attr->next) 
              ~~~~~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:731:16: note: 
      forward declaration of 'struct _ipp_s'
typedef struct _ipp_s ipp_t;            /**** IPP request/response data ****/
               ^
gtkprintbackendcups.c:1450:57: error: incomplete definition of type 'struct _ipp_attribute_s'
  for (attr = response->attrs; attr != NULL; attr = attr->next) 
                                                    ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1452:16: error: incomplete definition of type 'struct _ipp_attribute_s'
      if (!attr->name)
           ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1455:7: error: incomplete definition of type 'struct _ipp_attribute_s'
      _CUPS_MAP_ATTR_INT (attr, state, "job-state");
      ^                   ~~~~
gtkprintbackendcups.c:70:70: note: expanded from macro '_CUPS_MAP_ATTR_INT'
#define _CUPS_MAP_ATTR_INT(attr, v, a) {if (!g_ascii_strcasecmp (attr->name, (a))) v = attr->values[...
                                                                 ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1455:7: error: incomplete definition of type 'struct _ipp_attribute_s'
      _CUPS_MAP_ATTR_INT (attr, state, "job-state");
      ^                   ~~~~
gtkprintbackendcups.c:70:92: note: expanded from macro '_CUPS_MAP_ATTR_INT'
#define _CUPS_MAP_ATTR_INT(attr, v, a) {if (!g_ascii_strcasecmp (attr->name, (a))) v = attr->values[0].integ...
                                                                                       ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1450:32: warning: variable 'attr' used in loop condition not modified in loop body
      [-Wfor-loop-analysis]
  for (attr = response->attrs; attr != NULL; attr = attr->next) 
                               ^~~~
gtkprintbackendcups.c:1631:23: error: incomplete definition of type 'struct _ipp_s'
  for (attr = response->attrs; attr != NULL; attr = attr->next)
              ~~~~~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:731:16: note: 
      forward declaration of 'struct _ipp_s'
typedef struct _ipp_s ipp_t;            /**** IPP request/response data ****/
               ^
gtkprintbackendcups.c:1631:57: error: incomplete definition of type 'struct _ipp_attribute_s'
  for (attr = response->attrs; attr != NULL; attr = attr->next)
                                                    ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1699:34: error: incomplete definition of type 'struct _ipp_attribute_s'
      while (attr != NULL && attr->group_tag != IPP_TAG_PRINTER)
                             ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1700:20: error: incomplete definition of type 'struct _ipp_attribute_s'
        attr = attr->next;
               ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1705:34: error: incomplete definition of type 'struct _ipp_attribute_s'
      while (attr != NULL && attr->group_tag == IPP_TAG_PRINTER)
                             ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1707:25: error: incomplete definition of type 'struct _ipp_attribute_s'
        if (strcmp (attr->name, "printer-name") == 0 &&
                    ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1708:10: error: incomplete definition of type 'struct _ipp_attribute_s'
            attr->value_tag == IPP_TAG_NAME)
            ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1709:23: error: incomplete definition of type 'struct _ipp_attribute_s'
          printer_name = attr->values[0].string.text;
                         ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1710:23: error: incomplete definition of type 'struct _ipp_attribute_s'
        else if (strcmp (attr->name, "printer-uri-supported") == 0 &&
                         ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1711:8: error: incomplete definition of type 'struct _ipp_attribute_s'
                 attr->value_tag == IPP_TAG_URI)
                 ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1712:22: error: incomplete definition of type 'struct _ipp_attribute_s'
          printer_uri = attr->values[0].string.text;
                        ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
gtkprintbackendcups.c:1713:23: error: incomplete definition of type 'struct _ipp_attribute_s'
        else if (strcmp (attr->name, "member-uris") == 0 &&
                         ~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/cups/ipp.h:732:16: note: 
      forward declaration of 'struct _ipp_attribute_s'
typedef struct _ipp_attribute_s ipp_attribute_t;
               ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
7 warnings and 20 errors generated.
```
This is on 10.15/Xcode11.2, but also seen on 10.14.5/Xcode11.2. Does not fail on 10.14.5/Xcode10.3.

This looks similar to the gnomecups failure
https://sourceforge.net/p/fink/mailman/message/36803378/
and fixed for us in 7bc6c55

GTK+2 upstream removed cups <2.0 earlier this year, but their old code had cups1.6 support present. We might be able to copy the removed code to help us here.
https://github.com/GNOME/gtk/commit/3377133d1940ea4ab8f9d564e7e781bf74e51161#diff-f628c8c48a151a4017c0fe60d6c38ba1

--- a/10.9-libcxx/stable/main/finkinfo/gnome/gtk+2-10.15.info
+++ b/10.9-libcxx/stable/main/finkinfo/gnome/gtk+2-10.15.info
@@ -1,0 +1,339 @@
+Package: gtk+2
+Version: 2.18.9
+Revision: 100
+Distribution: 10.15
+Description: The Gimp Toolkit
+License: LGPL
+Maintainer: The Gnome Core Team <fink-gnome-core@lists.sourceforge.net>
+
+Depends: <<
+	%N-shlibs (= %v-%r),
+	glib2-shlibs (>= 2.22.0-1),
+	libgettext8-shlibs,
+	x11
+<<
+BuildDepends: <<
+	atk1 (>= 1.32.0-1),
+	cairo (>= 1.12.14-1),
+	fink (>= 0.24.12-1),
+	fink-package-precedence,
+	fontconfig2-dev (>= 2.10.0-1),
+	freetype219 (>= 2.6-1),
+	gettext-bin,
+	gettext-tools,
+	glib2-dev (>= 2.22.0-1),
+	gtk-doc (>= 1.13-5),
+	libgettext8-dev,
+	libjasper.1 (>= 1.900.1-4),
+	libjpeg9,
+	libpng16,
+	libtiff5,
+	pango1-xft2-ft219-dev (>= 1.24.5-4),
+	pkgconfig (>= 0.23),
+	x11-dev
+<<
+Conflicts: %N-common
+Replaces: %N-common
+Source: mirror:gnome:sources/gtk+/2.18/gtk+-%v.tar.bz2
+Source-MD5: f680d38c973635365acababacb1730ec
+#Source2: mirror:debian:pool/main/g/gtk+2.0/gtk+2.0_2.18.7-1.diff.gz
+Source2: https://ftp.gnome.org/mirror/archive/ftp.sunet.se/pub/mac/fink/gtk+2.0_2.18.7-1.diff.gz
+Source2-MD5: 4a41c6d2b5d29833759610abe1268686
+PatchFile: %n.patch
+PatchFile-MD5: ff65044275710a14327d9c813ab2fc4e
+PatchScript: <<
+#!/bin/sh -ev
+	sed -e 's|@PREFIX@|%p|g' <%{PatchFile} | patch -p1
+	gzip -dc ../gtk*.diff.gz | patch -p1
+	for file in \
+		015_default-fallback-icon-theme.patch \
+		042_treeview_single-focus.patch \
+		060_ignore-random-icons.patch \
+	; do
+		echo "applying $file"
+		patch -p1 < debian/patches/$file
+	done
+#	perl -pi -e 's,/usr/bin/env python,/usr/bin/python,' gtk/gtk-builder-convert
+	perl -pi -e 's,/usr/share/gtk-doc/html,%p/share/gtk-doc/html,g' docs/reference/*/html/*.html
+	perl -pi -e 's/\$\(no_undefined\)/-no-undefined/' `find . -name Makefile.in`
+	perl -pi -e 's,libpng12,libpng16,' configure
+<<
+
+SetCPPFLAGS: -D_IPP_PRIVATE_STRUCTURES=1
+SetCFLAGS: -Os
+SetCXXFLAGS: -Os
+ConfigureParams: <<
+	--mandir=%p/share/man \
+	--disable-shm \
+	--disable-gtk-doc \
+	--enable-dependency-tracking \
+	--disable-explicit-deps \
+	--disable-cups \
+	--disable-papi \
+	--with-libjasper \
+	--disable-visibility \
+	PKG_CONFIG_PATH="%p/lib/glib-2.0/pkgconfig-strict:$PKG_CONFIG_PATH"
+<<
+
+CompileScript: <<
+	./configure %c
+	make
+	fink-package-precedence --prohibit-bdep %n-dev .
+<<
+InfoTest: TestScript: make -C tests || exit 2
+InstallScript: <<
+	make install DESTDIR=%d
+	find %i/lib/gtk-2.0 -name \*.la | xargs perl -pi -e "s/^(dependency_libs|inherited_linker_flags)=.*/\1=''/" 
+
+	install -d -m 755 %i/sbin
+	install -c -m 755 update-gdk-pixbuf-loaders %i/sbin/
+	install -c -m 755 update-gtk-immodules %i/sbin/
+
+	# libversioned actual placed in %N-shlibs because needed
+	# during %N-shlibs PostInst; keep unversioned links in %N for
+	# legacy compat for others' PostInst
+	mv %i/bin/gdk-pixbuf-query-loaders     %i/bin/gdk-pixbuf-query-loaders-2.0
+	ln -s     gdk-pixbuf-query-loaders-2.0 %i/bin/gdk-pixbuf-query-loaders
+	perl -pi -e 's/(gdk-pixbuf-query-loaders)/\1-2.0/' %i/sbin/update-gdk-pixbuf-loaders
+	mv %i/sbin/update-gdk-pixbuf-loaders     %i/sbin/update-gdk-pixbuf-loaders-2.0
+	ln -s      update-gdk-pixbuf-loaders-2.0 %i/sbin/update-gdk-pixbuf-loaders
+	mv %i/sbin/update-gtk-immodules     %i/sbin/update-gtk-immodules-2.0
+	ln -s      update-gtk-immodules-2.0 %i/sbin/update-gtk-immodules
+
+	install -d -m 755 %i/etc/gtk-2.0
+	touch %i/etc/gtk-2.0/gtk.immodules
+	touch %i/etc/gtk-2.0/gdk-pixbuf.loaders
+<<
+DocFiles: <<
+	AUTHORS COPYING ChangeLog
+	gdk-pixbuf/ChangeLog:ChangeLog.gdk-pixbuf
+	modules/engines/pixbuf/ChangeLog:ChangeLog.modules.engines.pixbuf
+	modules/other/gail/ChangeLog:ChangeLog.modules.other.gail
+	NEWS README
+<<
+SplitOff: <<
+	Package: %N-shlibs
+	Depends: <<
+		atk1-shlibs (>= 1.32.0-1),
+		cairo-shlibs (>= 1.12.14-1),
+		dpkg-base-files,
+		fontconfig2-shlibs (>= 2.10.0-1),
+		freetype219-shlibs (>= 2.6-1),
+		glib2-shlibs (>= 2.22.0-1),
+		libgettext8-shlibs,
+		libjasper.1-shlibs (>= 1.900.1-4),
+		libjpeg9-shlibs,
+		libpng16-shlibs,
+		libtiff5-shlibs,
+		pango1-xft2-ft219-shlibs (>= 1.24.5-4),
+		pango1-xft2-ft219 (>= 1.24.5-4),
+		shared-mime-info (>= 0.70-1),
+		x11-shlibs
+	<<
+	Replaces: <<
+		%N (<< 2.14.4-1),
+		%N-dev (<< 2.18.7-1),
+		gtk2-engines (<< 2.6.3-1),
+		gail16, gail17, gail18,
+		gail18-shlibs
+	<<
+	Conflicts: jpilot (<< 0.99.9-12), jpilot-ssl, sylpheed-gpgme (<< 2.3.0-2), sylpheed (<< 2.3.0-2), sylpheed-ssl
+	DocFiles: <<
+		AUTHORS COPYING ChangeLog
+		gdk-pixbuf/ChangeLog:ChangeLog.gdk-pixbuf
+		modules/engines/pixbuf/ChangeLog:ChangeLog.modules.engines.pixbuf
+		modules/other/gail/ChangeLog:ChangeLog.modules.other.gail
+		NEWS README
+	<<
+	RuntimeVars: GDK_USE_XFT: 1
+	Files: <<
+		bin/gdk-pixbuf-query-loaders-2.0
+		bin/gtk-query-immodules-2.0
+		etc/gtk-2.0
+		lib/libgailutil.18*.dylib
+		lib/lib*-2.0.*.dylib
+		lib/gtk-2.0/2.10.0/engines
+		lib/gtk-2.0/2.10.0/immodules
+		lib/gtk-2.0/2.10.0/loaders
+		lib/gtk-2.0/2.10.0/printbackends
+		lib/gtk-2.0/modules
+		sbin/update-gdk-pixbuf-loaders-2.0
+		sbin/update-gtk-immodules-2.0
+		share/locale
+		share/man/man1/gtk-query-immodules-2.0.1
+		share/themes
+	<<
+	Shlibs: <<
+		%p/lib/libgailutil.18.dylib           19.0.0 %n (>= 2.14.3-1)
+		%p/lib/libgdk-x11-2.0.0.dylib         1801.0.0 %n (>= 2.18.7-1)
+		%p/lib/libgdk_pixbuf-2.0.0.dylib      1801.0.0 %n (>= 2.18.7-1)
+		%p/lib/libgdk_pixbuf_xlib-2.0.0.dylib 1801.0.0 %n (>= 2.18.7-1)
+		%p/lib/libgtk-x11-2.0.0.dylib         1801.0.0 %n (>= 2.18.7-1)
+	<<
+	PostInstScript: <<
+		case "$1" in
+			configure)
+				update-gdk-pixbuf-loaders-2.0
+				update-gtk-immodules-2.0
+				;;
+		esac
+	<<
+<<
+SplitOff2: <<
+	Package: %N-dev
+	Depends: <<
+		%N (= %v-%r),
+		%N-shlibs (= %v-%r),
+		glib2-shlibs (>= 2.22.0-1),
+		libgettext8-shlibs,
+		x11
+	<<
+	Conflicts: <<
+		gtk+ (<< 1.2.10-12),
+		gail16-dev, gail17-dev, gail18-dev
+	<<
+	Replaces: <<
+		%N (<< 2.14.5-1),
+		%N-shlibs (<< 2.14.3-1),
+		gail16, gail17, gail18,
+		gail16-dev, gail17-dev, gail18-dev
+	<<
+	BuildDependsOnly: True
+	DocFiles: <<
+		AUTHORS COPYING ChangeLog
+		gdk-pixbuf/ChangeLog:ChangeLog.gdk-pixbuf
+		modules/engines/pixbuf/ChangeLog:ChangeLog.modules.engines.pixbuf
+		modules/other/gail/ChangeLog:ChangeLog.modules.other.gail
+		NEWS README
+	<<
+	Files: <<
+		bin/gdk-pixbuf-csource
+		bin/gtk-builder-convert
+		include
+		lib
+		share/aclocal
+		share/gtk-doc
+		share/man/man1/gdk-pixbuf-csource.1
+		share/man/man1/gtk-builder-convert.1
+	<<
+<<
+
+Homepage: http://www.gtk.org
+DescDetail: <<
+The X libraries originally written for the GIMP, which are now used by
+several other programs as well.
+<<
+DescUsage: <<
+Packages which install gdk-pixbuf loader(s) should call update-gdk-
+pixbuf-loaders in their PostInstScript and PostRmScript to update the
+gdk-pixbuf.loaders file.
+
+Packages which install gtk immodule(s) should call update-gtk-immodules
+in their PostInstScript and PostRmScript to update the gtk.immodules file.
+
+Reinstall %N-shlibs to regenerate those index files if they get
+damaged or lost.
+
+If you get "Operation not supported" errors when trying to open URLs
+in gtk-based apps, try installing gvfs.
+<<
+
+DescPackaging: <<
+Older versions of gtk2-engines had a pixbuf theme, but now gtk+2 has
+that itself (and gtk2-engines has dropped or renamed it).
+
+	Move some .la %N-shlibs -> %N.dev as of 2.14.3-1
+
+	Minimize propagation of InheritedBuildDepends
+
+	The stand-alone gail packages are obsolete as of gtk+2 2.14.3-1,
+	at which point gtk+2 starting including gail18.
+
+	Move lib/gtk-2.0/modules/*.so %N -> %N-shlibs as of 2.14.3-2
+	(dir is as versioned as pkgname is)
+
+	Move etc/gtk-2.0/im-multipress.conf %N -> %N-shlibs as of 2.14.3-2
+	gtk-2.0/2.10.0/immodules/im-multipress.so in %N-shlibs
+	(dir is as versioned as pkgname is, could change name in
+	modules/input/gtkimcontextmultipress.c if need orthogonal
+	filename for new  libversion pkgname)
+
+	Move share/themes %N -> %N-shlibs as of 2.14.3-2
+	(dir is as versioned as pkgname is, could split off into
+	%N-themes if needed to keep across new libversion pkgname)
+
+	Move gdk-pixbuf-csource %N -> %N-dev as of 2.14.5-1
+	(developer tool to generate C code)
+
+	Patch gtk-builder-convert to work with any python version.
+	See: http://bugzilla.gnome.org/show_bug.cgi?id=559947
+
+	As of 2.14.7-1, %N-shlibs contains a libversioned form of many
+	utility programs that generate files needed by %N-shlibs so
+	that users don't need to install %N in order for %N-shlibs to
+	have the files. All data being handled is already in
+	libversioned directories, so still sane for Shlibs Policy.
+
+	As of 2.14.7-1, there are blank files installed in the .deb
+	for the live indexes generated by the update-gtk-immodules and
+	update-gdk-pixbuf-loaders scripts (run during PostInst).
+	Having these files "owned" allows dpkg to remove them
+	automatically when the package that can create them is
+	removed, rather than needing to play PostRm purge and hope
+	that doesn't interfere with a differently-named package that
+	may handle these files in the future.
+
+	Regenerating gtk-doc files The Right Way (--enable-gtk-doc) is
+	very slow, so just patch the prefix in the bad URLs in the
+	existing generated files.
+
+	Fix install of manpages without regenerating them. See:
+	http://bugzilla.gnome.org/show_bug.cgi?id=578765
+
+	Tentatively okay to ignore diagnostics about symbolic color
+	names in some themes' gtkrc. See:
+	http://bugzilla.gnome.org/show_bug.cgi?id=586711
+
+	Clear dep_libs from modules' .la because some inherit
+	-framework flag from apple's cups-config.
+
+	Don't lose early LIBS settings during ./configure. See:
+	https://bugzilla.gnome.org/show_bug.cgi?id=608558
+
+	Make sure we look for correct libpng (x11 may supply an older
+	one that confuses configure test)
+
+	Import modernization of libpng API jmpbuf calls from
+	gdk-pixbuf-2.23.5
+
+	"cups-config --libs" sometimes publishes "-liconv", which is
+	not part of the public API of cups, so elide it rather than
+	pushing a useless dep
+
+	pango runtime (pango1-xft2-ft129-shlibs) often whines about
+	missing fonts, which are in pango1-xft2-ft219. That package
+	cannot be made a dep of the -shlibs (would be circular). So we
+	have it as a dep here, which has the effect of "any package
+	using gtk+2 runtime gets pango's fonts at runtime, so pango's
+	runtime is happy"
+<<
+DescPort: <<
+	gtk+2-dev conflicts with gtk+ (<< 1.2.11), because it has
+	symbolic link which intercepts include files of the package.
+
+	Import some Debian patches (whichever ones look useful)
+
+	Make sure libgdk-x11 links against Xrender even on 10.4. See:
+	http://bugzilla.gnome.org/show_bug.cgi?id=555763
+
+	CUPS backend uses httpGetHostname(), which is only available
+	with cups>=1.2, which only comes with OS X >= 10.5. Use
+	upstream patch to import local copy of that function for
+	platforms where system cups is too old.
+	https://bugzilla.gnome.org/show_bug.cgi?id=610176
+
+	Hidden-visibility is only partially supported on non-ELF and
+	causes problems on 10.7 due to use of weak aliases. Removed in
+	future glib upstream, so scrap it here too. See:
+	http://permalink.gmane.org/gmane.os.apple.fink.gnome/2592
+<<

--- a/10.9-libcxx/stable/main/finkinfo/gnome/gtk+2.info
+++ b/10.9-libcxx/stable/main/finkinfo/gnome/gtk+2.info
@@ -1,6 +1,7 @@
 Package: gtk+2
 Version: 2.18.9
 Revision: 13
+Distribution: 10.9, 10.10, 10.11, 10.12, 10.13, 10.14, 10.14.5
 Description: The Gimp Toolkit
 License: LGPL
 Maintainer: The Gnome Core Team <fink-gnome-core@lists.sourceforge.net>

